README.md

```diff
@@ -156,33 +152,34 @@ to decide whether to apply it.
 
 ## Recommended reading
 
-To gain a better understanding of how Elastic CI Stack works and how to use it most effectively and securely, see the following resources:
+Following on from the [Getting started](#getting-started) pages above, to gain a more detailed understanding of how Elastic CI Stack works and how to use it most effectively and securely, see the following resources:
 
-* [Elastic CI Stack for AWS overview](https://buildkite.com/docs/agent/v3/elastic_ci_aws)
-* [Elastic CI Stack for AWS tutorial](https://buildkite.com/docs/tutorials/elastic-ci-stack-aws)
-* [Running Buildkite Agent on AWS](https://buildkite.com/docs/agent/v3/aws)
-* [Template parameters for Elastic CI Stack for AWS](https://buildkite.com/docs/agent/v3/elastic-ci-aws/parameters)
-* [Using AWS Secrets Manager](https://buildkite.com/docs/agent/v3/aws/secrets-manager)
-* [VPC Design](https://buildkite.com/docs/agent/v3/aws/vpc)
-* [CloudFormation Service Role](https://buildkite.com/docs/agent/v3/elastic-ci-aws/cloudformation-service-role)
+- [Template parameters for Elastic CI Stack for AWS](https://buildkite.com/docs/agent/v3/elastic-ci-aws/parameters)
+- [Using AWS Secrets Manager](https://buildkite.com/docs/agent/v3/aws/secrets-manager)
+- [VPC Design](https://buildkite.com/docs/agent/v3/aws/vpc)
+- [CloudFormation Service Role](https://buildkite.com/docs/agent/v3/elastic-ci-aws/cloudformation-service-role)
 
 ## Questions and support
 
-Feel free to drop an email to support@buildkite.com with questions. It helps us if you can provide the following details:
+Feel free to drop an email to support@buildkite.com with questions. It'll also help us if you can provide the following details:
```

Comment:
I'm OK with sticking to 'us' in this context, since it's pretty clear from the previous sentence that the 'us' refers to 'Buildkite'.